varya/assets/sass/blocks/search/_style.scss

```diff
@@ -0,0 +1,11 @@
+.wp-block-search {
+    display: flex;
+    max-width: var(--responsive--aligndefault-width);
```

Comment:
Making an assumption here that the search block isn't going to go beyond wide or full widths.

Reply:
Yeah, I think this assumption is perfectly fine.